src/renderers/common/extras/PMREMGenerator.js

```diff
@@ -318,9 +318,7 @@ class PMREMGenerator {
 
 	_compileMaterial( material ) {
 
-		const tmpMesh = this._lodMeshes[ 0 ];
```

Comment:
`this._lodMeshes[ 0 ]` is undefined because `_allocateTargets()` has not been executed yet. Does the compile still produce the expected result if a temporary mesh is used like below?

Reply:
It seems no since the render object will be different when a temporary mesh is used. It seems `_allocateTargets()` (or at least `_createPlanes()` ) must run before a compile happens.

Reply:
Maybe we can have a mesh/geometry just to compile this shader, ignoring `_allocateTargets()` for now, it's needs to be relative with the size of texture.

I think we will need to add `this._compileMaterial( this._blurMaterial )` too

Reply:
I'll merge the PR for the moment so the severe issues are fixed. The code in `_compileMaterial()` is now aligned to the old `PMREMGenerator` so we might want to improve both at the same time.